src/components/calculator/PointsDisplay.tsx

```diff
@@ -29,49 +35,296 @@ function displayPercent(points: number): string {
+
+  const personRiskEachFormatted = personRiskEach.toLocaleString()
+  const pointsFormatted = displayPoints(points)
+  const pointsPercentFormatted = displayPercent(points)
+  const activityRiskFormatted = fixedPointPrecisionPercent(activityRisk)
+  const personCountSuffixFormatted =
+    props.data.personCount === 1 ? 'person' : 'people'
+  const frequencyFormatted = props.repeatedEvent
+    ? 'per week'
+    : 'each time you do it'
+  const lowerBoundFormatted = displayPoints(props.lowerBound)
+  const upperBoundFormatted = displayPoints(props.upperBound)
+  const budgetFormatted = displayPoints(props.riskBudget)
+  const budgetAnnualPercentFormatted = fixedPointPrecisionPercent(
+    props.riskBudget / 1000000,
+  )
+  const weekBudgetFormatted = getWeekBudget(props.riskBudget)
+  const budgetConsumptionFormatted = budgetConsumption(
+    props.points,
+    props.riskBudget,
+  )
+
+  const calculationBreakdown = (
+    <>
+      <span
+        className="expandable-header"
+        onClick={() =>
+          setCalculationExplanation(!displayCalculationExplanation)
+        }
+        aria-controls="calculation-explanation"
+        aria-expanded={displayCalculationExplanation}
+      >
+        {displayCalculationExplanation ? (
+          <>
+            <BsChevronDown /> How this was calculated
+          </>
+        ) : (
+          <>
+            <BsChevronRight /> Learn how this was calculated
+          </>
+        )}
+      </span>
+      <Collapse in={displayCalculationExplanation}>
+        <div id="calculation-explanation">
+          <div>
+            <h4>Calculation:</h4>
+            <div id="calculation-breakdown">
+              <code>
+                ({personRiskEachFormatted} Person Risk) x (
+                {activityRiskFormatted} Activity Risk) x (
+                {props.data.personCount} {personCountSuffixFormatted})<br />
+                <strong style={{ fontSize: '1.5em' }}>
+                  = ~{pointsFormatted} microCOVIDs {frequencyFormatted}
+                </strong>{' '}
+                <br />
+                (margin of error: {lowerBoundFormatted} to {upperBoundFormatted}
+                )
+              </code>{' '}
+            </div>
+          </div>
+          <h4>Calculation Steps:</h4>
+          <ol>
+            <li>
+              <strong>
+                Person Risk:{' '}
+                <code>{personRiskEachFormatted}-in-a-million chance</code>
+              </strong>{' '}
+              <br />
+              First, we calculate that each other person in this area has a{' '}
+              <strong>{personRiskEachFormatted}-in-a-million chance</strong>
+              {'  '} of currently having COVID.
+            </li>
+            <li>
+              <strong>
+                Activity Risk: <code>{activityRiskFormatted} chance</code>
+              </strong>
+              <br /> Next, we calcualte the risk of the activity. Assuming 1
```

Comment:
"Assuming 1 person at this activity has COVID" -> "A person at this activity with COVID would have a X chance of transmitting it to you"

I find the "assuming" language to be confusing.